components/VideoEmbedPreview/VideoEmbedPreview.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { Meta, StoryObj } from "@storybook/react";
+import VideoEmbedPreview from "./VideoEmbedPreview";
+
+const meta = {
+  title: "VideoEmbedPreview",
+  component: VideoEmbedPreview,
+  args: {
+    video: {
+      url: "https://www.youtube.com/watch?v=eoGSR7kNARE",
+    },
+  },
+} satisfies Meta<typeof VideoEmbedPreview>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  parameters: {
+    viewport: {
+      defaultViewport: "responsive",
+    },
+  },
+};
+
+export const Desktop: Story = {
+  parameters: {
+    viewport: {
+      defaultViewport: "desktop",
+    },
+  },
+};
```

Comment:
These two (Default and Desktop) look exactly the same, let's get rid of one (I'm thinking "desktop").
Other than that. It looks great!